src/test/java/com/adyen/httpclient/ClientTest.java

```diff
@@ -41,7 +50,7 @@ public void testConfigLiveClient() {
         config.setLiveEndpointUrlPrefix("prefix");
         config.setApiKey(apiKey);
         Client client = new Client(config);
-        Assert.assertEquals(Environment.LIVE, client.getConfig().getEnvironment());
+        assertEquals(Environment.LIVE, client.getConfig().getEnvironment());
```

Comment:
boys scout rule: should we also assert that the prefix is correctly set? We can add another test here